src/components/utils/TimeUtils.js

```diff
@@ -1,9 +1,39 @@
-import moment from 'moment'
+import format from 'date-fns/format'
+
+const seconds = n => 1000 * n
+const minutes = n => seconds(60) * n
+const hours = n => minutes(60) * n
+const days = n => hours(24) * n
+
+function unix(time) {
```

Comment:
Oh, because of the need for methods to call each other, and using 'this' won't always work due to how we're calling the methods, sometimes it's undefined. If you have another way you'd like to handle this, that's great, I wasn't thrilled about this method.